packages/legacy-client/src/client/baseTypes/geoshapes/index.ts

```diff
@@ -14,7 +14,11 @@
  * limitations under the License.
  */
 
+// eslint seems to incorrectly assume that `export * as Distance from "./DistanceMethods"` in Distance.ts exports the individual methods by name
+/* eslint-disable import/export */
 export * from "./Distance";
+export * from "./DistanceMethods";
+/* eslint-enable import/export */
```

Comment:
^ this is a weird eslint thing, we should probably file a bug report here at some point